app/src/main/java/mozilla/org/webmaker/activity/Page.java

```diff
@@ -7,4 +7,9 @@
     public Page() {
         super("page", R.id.page_layout, R.layout.page_layout, R.menu.menu_page);
     }
+
+    @Override
+    public void onBackPressed() {
```

Comment:
We've got a bunch of places where we need to override the back button ... moving it into `WebmakerActivity` I think is a good idea.